### DISCUSSION TITLE
My project not working

### DISCUSSION THREAD

**Discussion by ateeqbhatti** [GrapesJS/grapesjs#5028] [Q&A]
Hey everyone, i was using grape js in my final year project that is in the upcoming week, but they changed the APIs yesterday that led to the breakage of my project
<link rel="stylesheet" href="stylesheets/toastr.min.css">
<link rel="stylesheet" href="stylesheets/grapes.min.css?v0.20.3">
<link rel="stylesheet" href="stylesheets/grapesjs-preset-webpage.min.css">
<link rel="stylesheet" href="stylesheets/tooltip.css">
<link href="https://unpkg.com/grapick/dist/grapick.min.css" rel="stylesheet">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<script src="js/toastr.min.js"></script>
<script src="js/grapes.min.js?v0.20.3"></script>
<script src="https://unpkg.com/grapesjs-preset-webpage@1.0.2"></script>
<script src="https://unpkg.com/grapesjs-blocks-basic@1.0.1"></script>
<script src="https://unpkg.com/grapesjs-plugin-forms@2.0.5"></script>
<script src="https://unpkg.com/grapesjs-component-countdown@1.0.1"></script>
<script src="https://unpkg.com/grapesjs-plugin-export@1.0.11"></script>
<script src="https://unpkg.com/grapesjs-tabs@1.0.6"></script>
<script src="https://unpkg.com/grapesjs-custom-code@1.0.1"></script>
<script src="https://unpkg.com/grapesjs-touch@0.1.1"></script>
<script src="https://unpkg.com/grapesjs-parser-postcss@1.0.1"></script>
<script src="https://unpkg.com/grapesjs-tooltip@0.1.7"></script>
<script src="https://unpkg.com/grapesjs-tui-image-editor@0.1.3"></script>
<script src="https://unpkg.com/grapesjs-typed@1.0.5"></script>
<script src="https://unpkg.com/grapesjs-style-bg@2.0.1"></script> 
<link rel="stylesheet" href="stylesheets/toastr.min.css">
i was using the following linnks , but now my editor is not working, please help

**Comment by jcenlo** [comment]
The best solution for you i think is use the previous version but i think are you using already 
`<script src="js/grapes.min.js?v0.20.3"></script>`

I think API is not the problem, could be one of this scripts i think you should remove all that PLUGIN's scripts except grapesjs and add one by one  to see where is the problem

you can download previous version from here
https://github.com/GrapesJS/grapesjs/releases/tag/v0.20.4
or
https://www.npmjs.com/package/grapesjs/v/0.20.4
npm i grapesjs@0.20.4
